sample-211/src/main/java/com/infullmobile/android/infullmvp/sample/dagger11/sample_mvp_activity/SampleMvpActivityModel.kt

```diff
@@ -0,0 +1,9 @@
+package com.infullmobile.android.infullmvp.sample.dagger11.sample_mvp_activity
+
+import javax.inject.Inject
+
+class SampleMvpActivityModel @Inject constructor() {
+
+    val displayMessage = ""
```

Comment:
displayMessage? maybe: messageToBeDisplayed? or just message?

Besides, why is it empty? can't we add any kind of test to it? It's a sample Yo! It's ought to be visualising something. Not just blank screens.